src/test/java/com/databricks/jdbc/api/impl/converters/ByteConverterTest.java

```diff
@@ -76,7 +76,7 @@ public void testConvertToChar() {
 
   @Test
   public void testConvertToString() throws DatabricksSQLException {
-    assertEquals(new ByteConverter().toString(NON_ZERO_OBJECT), "A");
+    assertEquals(new ByteConverter().toString(NON_ZERO_OBJECT), "65");
   }
```

Comment:
I may be skeptical here (and just so we don't add a breaking change later)

Can you add a more elaborate test for negative boundary values too? 


```suggestion
  @ParameterizedTest
  @CsvSource({
      "-128, -128",
      "-1, -1",
      "0, 0",
      "1, 1",
      "10, 10",
      "65, 65",
      "127, 127"
  })
  public void testConvertToString(byte input, String expected) throws DatabricksSQLException {
      assertEquals(new ByteConverter().toString(input), expected);
  }
```